people/templates/registration/logged_out.html

```diff
@@ -0,0 +1,7 @@
+{% extends 'base.html' %}
```

Comment:
```suggestion
{% extends 'main.html' %}
```
We should end up extending main instead of base for all of these templates at some point so we get the navigation and other elements of the base UI design, however it's not the end of the world if that isn't implemented now as it's easy to change later

Reply:
You are right, I was not really concern about frontend at time because Noah was still working on templates and I did not know what the right structure would be